tests/entrypoints/openai/test_prompt_validation.py

```diff
@@ -0,0 +1,22 @@
+# imports for guided decoding tests
+import re
+
+import openai
+import pytest
+
+from ...utils import RemoteOpenAIServer
+
+
+@pytest.mark.asyncio
+async def test_empty_prompt():
+    model_name = "gpt2"
+    server_args = ["--disable-frontend-multiprocessing", "--enforce-eager"]
```

Comment:
Do you need to disable the front end multiprocessing? It would be nice to test through that too

Reply:
No, it was just to reduce the number of moving pieces. But I've enabled it again.